source/two_phase_base.cc

```diff
@@ -1097,6 +1101,8 @@ compute_bubble_statistics(std::vector<Tensor<2,3> > *,
     data[3+d] = global_velocity[d]/global_volume;
   for (unsigned int d=0; d<dim; ++d)
     data[3+dim+d] = global_mass_center[d]/global_volume;
+  data[3+dim+dim] = (std::pow(pi, 1./3.)
+		     * std::pow(6*global_volume, 2./3.)) / (global_area);
```

Comment:
Yes, that's what I realized also. At first I increased the size of data to 4+2*dim+1 but then I realized the two last places in data were not used before, so I used one of thous to store the sphericity instead.  (I don't know why data is too big?)